pandas/tests/indexes/period/test_setops.py

```diff
@@ -120,10 +120,6 @@ def test_union_misc(self, sort):
         with pytest.raises(period.IncompatibleFrequency):
             index.union(index2, sort=sort)
 
-        msg = 'can only call with other PeriodIndex-ed objects'
```

Comment:
do we have cases where we *would* raise *ever*?

Reply:
I do not believe so. Only the incompatible frequency error which is covered in the line above.